multiboot2/src/builder/information.rs

```diff
@@ -287,72 +287,126 @@ impl InformationBuilder {
         Self::build_add_bytes(bytes, &EndTag::default().struct_as_bytes(), true);
     }
 
-    pub fn basic_memory_info_tag(&mut self, basic_memory_info_tag: BasicMemoryInfoTag) {
-        self.basic_memory_info_tag = Some(basic_memory_info_tag)
+    /// Adds a 'basic memory information' tag (represented by [`BasicMemoryInfoTag`]) to the builder to later build the
```

Comment:
nit: "to the builder to later build the boot information from"

I think this is obvious, as it is a builder :)

Reply:
You are totally right. :D
I removed the respective parts.